.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
```

Comment:
Do you suggest that it's linked within an html comment? I think if it's not then you'll see a bunch of issues with "Please read how to contribute prior to submitting an issue" or whatever verbiage prior to the actual issue. Could get annoying.